augmenting job triggers page (#2944)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/job-triggers.md

---
title: "Job triggers"
id: "job-triggers"
description: "You can configure when and how dbt should run your job"
---

> In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should run your job:

Reviewer: ```suggestion
In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should [run your job](/docs/deploy/job-triggers#schedule-job):
```

- **Schedule** tab &mdash; Use the **Run on schedule** toggle to configure your job to run on either [scheduled days](#schedule-days) or [custom cron-powered schedule](#custom-cron-schedule)
- **Continuous Integration (CI)** tab &mdash; Configure [continuous integration](/docs/deploy/cloud-ci-job) to run when someone opens a new pull request in your dbt repository
- **API** tab &mdash; Use the [API](/docs/dbt-cloud-apis/admin-cloud-api) to trigger a job or send events to other systems

<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" width="85%" title="Configuring your job triggers"/>

## Schedule jobs

> 

Reviewer: ```suggestion
```

description: "You can use cron syntax to specify when you want to run a job."
---

In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should run your job:

- **Schedule** tab &mdash; Use the **Run on schedule** toggle to configure your job to run on either simple [scheduled days](#schedule-days) or [custom cron-powered schedules](#custom-cron-schedule)
- **Continuous Integration (CI)** tab &mdash; Configure [continuous integration](/docs/deploy/cloud-ci-job) to run when someone opens a new pull request in your dbt repository
- **API** tab &mdash; Use the [API](/docs/dbt-cloud-apis/overview) to trigger a job or send events to other systems

<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" title="Configuring your job triggers"/>

## Schedule jobs

> To schedule your job to run at specific days, times, and intervals, go to the **Schedule** tab, and toggle **Run on schedule.** Use either the [scheduled days](#schedule-days or the [custom cron-powered schedule](#custom-cron-schedule] method to customize your desired days, times, and intervals.

Author: ```suggestion
To schedule your job to run at specific days, times, and intervals:
1. Go to the specific job settings, click **Edit**
2. Then go to the **Triggers** section 
3. Go to the **Schedule** tab, and toggle **Run on schedule.** 
4. Use either the [scheduled days](#schedule-days or the [custom cron-powered schedule](#custom-cron-schedule] method to customize your desired days, times, and intervals.
```

description: "You can use cron syntax to specify when you want to run a job."
---

> In dbt Cloud, you can configure when and how dbt should trigger your job run.

Author: ```suggestion
In dbt Cloud, you can use the options under **Triggers** to configure when and how dbt should run your job:
```

- **Schedule** tab &mdash; Use the **Run on schedule** toggle to configure your job to run on either simple [scheduled days](#schedule-days) or [custom cron-powered schedules](#custom-cron-schedule)
- **Continuous Integration (CI)** tab &mdash; Configure [continuous integration](/docs/deploy/cloud-ci-job) to run when someone opens a new pull request in your dbt repository
- **API** tab &mdash; Use the [API](/docs/dbt-cloud-apis/overview) to trigger a job or send events to other systems

<Lightbox src ="/img/docs/dbt-cloud/using-dbt-cloud/triggers.jpg" title="Configuring your job triggers"/>

> ## Scheduling

Author: sugg clearer title
```suggestion
## Schedule jobs
```

To configure your job to run on particular day(s) and time(s), you can toggle the **Run on schedule** toggle and customize the days, time, and intervals you want your job to run.

> ### Schedule Days

Author: ```suggestion
### Schedule days
```

> To configure your job to run on particular day(s) and time(s), you can toggle the **Run on schedule** toggle and customize the days, time, and intervals you want your job to run.

Author: ```suggestion

To schedule your job to run at specific days, times, and intervals, go to the **Schedule** tab, and toggle **Run on schedule.** Use either the [scheduled days](#schedule-days or the [custom cron-powered schedule](#custom-cron-schedule] method to customize your desired days, times, and intervals.
```

> Under **Timing**, you can configure your job to run either at customizable hours hours or exact intervals. Customizable hours is useful for jobs that need to run frequently throughout the day, while at exact intervals is useful for jobs that need to run at specific times of day.

Author: ```suggestion
To set your job's schedule, use the **Schedule Days** option to choose specific days of the week, and select customized hours or intervals. 

Under **Timing**, you can either use customizable hours for jobs that need to run frequently throughout the day or exact intervals for jobs that need to run at specific times.
```

> #### "Every N hours"

Author: ```suggestion
- **Every `n` hours** &mdash; Use this option to set how often your job runs, in hours. Enter a number between 1 and 23 to represent the interval between job runs. For example, if you set it to "every 2 hours", the job will run every 2 hours from midnight UTC. This option is useful if you need to run jobs multiple times per day at regular intervals.
```

> This option allows you to set a frequency for your job to run, based on the number of hours between each run. You can enter a number between 1 and 23, which represents the interval between each job run. For example, if you set it to "every 2 hours," the job will run every 2 hours starting from midnight UTC. This option is useful if you want to run your jobs at fixed intervals throughout the day and have them run more frequently than once a day.

Author: ```suggestion

```

> #### "At exact intervals"

Author: ```suggestion
- **At exact intervals** &mdash; Use this option to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.
```

> This option allows you to set specific times when your job should run. You can enter a comma-separated list of hours (in UTC) when you want the job to run. For example, if you set it to `0,12,23,` the job will run at midnight, noon, and 11 PM UTC. This option is useful if you want your jobs to run at specific times of day and don't need them to run more frequently than once a day.

Author: ```suggestion

```

:::info

> dbt Cloud uses [coordinated universal time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time), it is not translated to your timezone or aware of daylight savings time. For examples:

Author: ```suggestion
dbt Cloud uses [Coordinated Universal Time](https://en.wikipedia.org/wiki/Coordinated_Universal_Time) (UTC) and does not account for translations to your specific timezone or take into consideration daylight savings time. For example:
```

- 0 means 12am (midnight) UTC
- 12 means 12pm (afternoon) UTC
- 23 means 11pm UTC

:::

> ### Custom Cron Schedule

Author: ```suggestion
### Custom cron schedule
```

> In dbt Cloud, you can use "cron" syntax to specify when you'd like your job to run. Cron syntax is very customizable, including the minute, hour, day of the month, month, and day of the week. This allows you to schedule jobs to run at specific times on specific days, as well as to set up more complex schedules, such as running a job on the first Monday of every month.

Author: ```suggestion
To fully customize the scheduling of your job, choose the **Custom cron schedule** option and use the "cron" syntax. With this syntax, you can specify the minute, hour, day of the month, month, and day of the week, allowing you to set up complex schedules like running a job on the first Monday of each month.
```

> If you need help coming up with the right cron syntax, we recommend using a tool like [crontab.guru](https://crontab.guru). There, you can enter cron snippets and see what they mean in plain English. You can also find some example snippets below.

Author: ```suggestion
Use tools such as [crontab.guru](https://crontab.guru/) to generate the correct cron syntax. This tool allows you to input cron snippets and returns their plain English translations. Refer to the following sample snippets:
```

> ### Examples

Author: ```suggestion

```

6. Enter the custom cron syntax for the schedule you want.

<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/job-schedule.png" title="Schedule your dbt job"/>

> THE BELOW IS JUST COPIED OVER TEXT - WILL AMEND SOON!

Reviewer 2: @mirnawong1 this will need some help, I added back in Jobs but not sure what else you had in mind :)

6. Select **Save**, then click **Run Now** to run your job. Click the run and watch its progress under "Run history." 

## Related docs

> - [Job Commands](/docs/deploy/job-commands)

Author: ```suggestion
- [Job commands](/docs/deploy/job-commands)
```
